src/Sarif.ValidationTests/Sarif.ValidationTests.csproj

```diff
@@ -26,6 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Sarif.Multitool\Sarif.Multitool.csproj" />
```

Comment:
>Multitool [](start = 56, length = 9)

Why?

Reply:
Honestly can't account for it. I was required to add this reference when I took a dependency on the private Jschema code i checked into multitool. After removing that (and doing other work) I was unable to remove this reference, either from the IDE UI or by editing the csproj. I must have somehow introduced a new dependency on this project. Will look for the relevant code in other changes. I need a resolution of 'beats me' to close this comment, choosing another less precise option instead.

---
In reply to: [255307688](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255307688) [](ancestors = 255307688)

Reply:
If you remove it, what's the diagnostic on the build break?

---
In reply to: [255312082](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255312082) [](ancestors = 255312082,255307688)